src/FSharp.Analyzers.Cli/Program.fs

```diff
@@ -23,7 +23,7 @@ let mutable verbose = false
 
 let createFCS () =
     let checker = FSharpChecker.Create(projectCacheSize = 200, keepAllBackgroundResolutions = true, keepAssemblyContents = true)
-    checker.ImplicitlyStartBackgroundWork <- true
+    // checker.ImplicitlyStartBackgroundWork <- true
```

Comment:
`ImplicitlyStartBackgroundWork` no longer exists.  Not sure what action to take here.

Reply:
That's fine - it was part of removing the reactor thread, so it's safe to remove